test/test_train_imagenet.py

```diff
@@ -71,10 +72,24 @@
 }
 
 
+def get_scheduler_for_model(model_name, optimizer, num_steps_per_epoch):
```

Comment:
* It should be possible to get rid of this function and use `MODEL_PROPERTIES` to get the scheduler.  This is how the other model specific parameters are obtained
```python
MODEL_PROPERTIES['resnet50']['lr_scheduler'] = WarmupScheduler(...)
```

* Better alternative is to have args for scheduler:
```bash
python test_train_imagenet.py \
 --model resnet50 \
 --lr-scheduler='warmupandexponentialdecay' \
 --lr-scheduler-divide-every-n-epochs=5 \
 --lr-scheduler-divisor=5 \ 
```

You can then parse this and construct scheduler all inside `test_utils`, reducing clutter here..

Reply:
I moved the scheduler params to FLAGS and got rid of this method. Now there is a more generic helper method in schedulers.py that doesn't encode any knowledge about the models of the caller